adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -847,6 +942,51 @@ func createRoute(params *routeCreateParams) *routev3.Route {
 				IdleTimeout:       ptypes.DurationProto(time.Duration(config.Envoy.Upstream.Timeouts.RouteIdleTimeoutInSeconds) * time.Second),
 			},
 		}
+		if config.Envoy.RateLimit.Enabled && ratelimitLevel != "" {
+			basePath += resourcePathParam
```

Comment:
this var basePath is used outside this if block right? so without changing it, shall we assign this to a new var